Code/src/main/java/nl/utwente/viskell/ui/components/DefinitionBlock.java

```diff
@@ -147,5 +148,16 @@ public void invalidateVisualState() {
     public boolean belongsOnBottom() {
         return true;
     }
+    
+    public LambdaContainer getBody() {
+        return body;
+    }
 
+    @Override
+    public void relocate(double x, double y) {
+        double dx = x-getLayoutX(), dy = y-getLayoutY();
+        super.relocate(x, y);
+        
+        body.moveNodes(dx, dy);
+    }
```

Comment:
Hopefully no container will behave this way since it's an artifact of the children not being children in the scene graph.
CBA to extract a class for this one method.